inc/admin/ui/modules/support.php

```diff
@@ -60,8 +60,8 @@
 		array(
 			'type'         => 'helper_help',
 			'description'  =>
-			/* translators: line break recommended, but not mandatory  */
-			__( '<strong>Now be specific!</strong><br>We have pre-filled the form with some questions for you to phrase your description along.', 'rocket' )
+			/* translators: line breaks recommended, but not mandatory  */
+			__( '<strong>Now be specific!</strong><br>We have pre-filled the form with some questions for you to phrase your description along.<br>We speak English, French, German, Italian, Serbian, and Spanish.', 'rocket' )
```

Comment:
Have I missed an update of the WP coding standard guidelines? 😳  
I may have, it’s been a while since I looked.